src/types/Time.ts

```diff
@@ -0,0 +1,29 @@
+import * as TypeUtils from '../TypeUtils'
+
+class Time implements SBVRType<string, Date> {
+	types = {
+		  postgres: 'TIME'
+		, mysql: 'TIME'
+		, websql: 'TEXT'
+		, odata: {
+			name: 'Edm.DateTime'
+		}
+	}
+
+	fetchProcessing = (data:string, callback:Callback<Date>) => {
+		// We append the date of the epoch so that we can parse this as a valid date.
+		let date = new Date('Thu, 01 Jan 1970 ' + data)
```

Comment:
Does it make sense to use more formal ISO format instead? `1970-01-01T${data}Z` or something along these lines?